indi-qhy/qhy_ccd.cpp

```diff
@@ -1778,6 +1840,50 @@ bool QHYCCD::ISNewSwitch(const char *dev, const char *name, ISState *states, cha
             IDSetSwitch(&AMPGlowSP, nullptr);
             return true;
         }
+
+        //////////////////////////////////////////////////////////////////////
+        /// Overscan Area
+        //////////////////////////////////////////////////////////////////////
+        //NEW CODE - Add support for overscan/calibration area 
+        else if (!strcmp(OverscanAreaSP.name, name))
+        {   
+            
+            IUUpdateSwitch(&OverscanAreaSP, states, names, n);
+            bool isIgnored = (OverscanAreaS[INDI_ENABLED].s == ISS_OFF); //Overscan Area included switch is 'off', thus excluded
+            
+            if (isIgnored == IgnoreOverscanArea)
+                return true;
```

Comment:
Before "return true" you must update the property, otherwise, it would remain as IPS_BUSY, so

if (isIgnored == IgnoreOverscanArea)
{
   OverscanAreaSP.s = IPS_OK;
   IDSetSwitch(&OverscanAreaSP, nullptr);
   return true;
}

Reply:
Also updated..